### DISCUSSION TITLE
Subscription Link

### DISCUSSION THREAD

**Discussion by akv79** [hiddify/Hiddify-Manager#189] [Q&A سوالات رایج] [answered]
سلام خسته نباشید
بخش های Subscription Link و Subscription Link b64 که تو صفحه کانفیگ ها هست دقیقا چی هستن؟

**Comment by zZedix** [comment, marked as answer]
سابسکریبشن لینک تمامی کانکشن هارو به صورت یکجا روی نرم افزار هایی که پشتیبانی میکنند مثل fair vpn اضافه میکنه . b64 همونه ولی کد گذاری شده

**Comment by ehsundesign** [comment]
برای آیفون از b64 باید استفاده بشه
